app/models/ldap.rb

```diff
@@ -29,11 +31,15 @@ def attributes(result)
     end
 
     def default_connection
-      @default_connection ||= Net::LDAP.new host: "ldap.princeton.edu", base: "o=Princeton University,c=US", port: 636,
-                                            encryption: {
-                                              method: :simple_tls,
-                                              tls_options: OpenSSL::SSL::SSLContext::DEFAULT_PARAMS
-                                            }
+      Net::LDAP.new(
```

Comment:
Why remove the memoizing here?  I'm unsure why we would want to create this connection multiple times

Reply:
I'm not sure how this happened, maybe a rebase issue? I'll re-memoize.